game/scripts/npc/abilities/ember_spirit_sleight_of_fist.txt

```diff
@@ -18,7 +18,7 @@
     "MaxLevel"                                            "6"
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "18 14 10 6 6 6"
+    "AbilityCooldown"                                     "12 10 8 6 6 6" //OAA
```

Comment:
This was buffed recently by Valve and I think this is the only thing about Ember Spirit that doesn't need a buff.

Reply:
When I say the only thing that doesn't need a buff its the Sleight of Fist cooldown. You already buffed creep_damage_penalty and radius (which I don't like). Cooldown and damage bonus against heroes were buffed recently by Valve already.

Reply:
I'll revert the radius but these changes cause embers who are core 1 to be able to stay up to speed with other meta core 1's. 18 second cd for a farming ability to do -50% dmg to creeps at level 1 is a feelsbadman in my opinion, you could probably convince me otherwise but that's the reason behind the buffs.